pkgs/development/python-modules/ale-py/default.nix

```
@@ -0,0 +1,93 @@
+{ buildPythonPackage
+, SDL2
+, cmake
+, fetchFromGitHub
+, gcc
+, git
+, gym
+, importlib-metadata
+, importlib-resources
+, lib
+, ninja
+, numpy
+, pybind11
+, pytestCheckHook
+, python
+, pythonOlder
+, setuptools
+, typing-extensions
+, wheel
+, zlib
+}:
+buildPythonPackage rec {
+  pname = "ale-py";
+  version = "0.8.0";
+  format = "pyproject";
+  rev-short = "d59d006";
+
+  src = fetchFromGitHub {
+    owner = "mgbellemare";
+    repo = "Arcade-Learning-Environment";
+    rev = "v${version}";
+    sha256 = "sha256-OPAtCc2RapK1lALTKHd95bkigxcZ9bcONu32I/91HIg=";
+  };
+
+  buildInputs = [
+    zlib
+    SDL2
+    setuptools
+    wheel
+    pybind11
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    gcc
+  ];
```

```suggestion
  nativeBuildInputs = [
    cmake
    gcc
  ];

  buildInputs = [
    zlib
    SDL2
    setuptools
    wheel
    pybind11
  ];
```

Isn't pybind11, wheel and setuptools run on the build machine? Then they all go into nativeBuildInputs.

Also does this always require gcc? Then we should use gccStdenv.